### PR TITLE
Add Spanish NLP

### PR DESCRIPTION
**Pull request by RichardLitt**
Added @dav009's list.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -343,6 +343,7 @@ Please read the [contribution guidelines](contributing.md) or the [creating a li
 - [Linux Containers](https://github.com/Friz-zy/awesome-linux-containers)
 - [Beginner-Friendly Projects](https://github.com/MunGell/awesome-for-beginners)
 - [Bluetooth Beacons](https://github.com/beaconinside/awesome-beacon)
+- [Spanish NLP](https://github.com/dav009/awesome-spanish-nlp)
```
Comment (sindresorhus) [inline readme.md]: I don't think NLP is a common enough acronym to use here, can you use the full form instead?

**Comment by sindresorhus** [comment]
Looks interesting, but the list doesn't look very active.

**Comment by sindresorhus** [comment]
@dav009 Can you read the [contributing guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [awesome manifest](https://github.com/sindresorhus/awesome/blob/master/awesome.md), and ensure your list follows it? Mainly, license, contributing guidelines, description for items. I would also clarify the headings `POS TAGGERS`, `MLWE`, `NER`. Not everyone interested knows the lingo.

File: readme.md (hunk 1 above)
Comment (tastyminerals) [inline readme.md]: it is, NLP is a common enough acronym for Natural Language Processing and using anything different will upset a lot of NLP ppl :)

Comment (sindresorhus) [inline readme.md]: Acronyms are always common and well known for its target group, but it's not very approachable for outside people that might be interested. Never hurts to be explicit.

Comment (sindresorhus) [inline readme.md]: Example: LGTM (Looks good to me) and PR (Pull request) are so natural for me, but I constantly see people being confused by them.

Comment (RichardLitt) [inline readme.md]: Hmm. I'm not sure. NLP is more common, to me, than LGTM. The only thing that might make it confusing is NeuroLinguistic Programming has the same acrostic. I can spell it out and add it, though.

**Comment by dav009** [comment]
@RichardLitt awesome.
@sindresorhus Gonna make  those changes and come back to this PR

**Comment by RichardLitt** [comment]
<3

**Comment by dav009** [comment]
@sindresorhus  updated:
 - Added contribution guidelines
 - License
 - Expanded Acronyms

**Comment by sindresorhus** [comment]
Added. Would still like to see more descriptions for the items, though.

**Pull request closed without merging**